stylesheet/style.css

```diff
@@ -587,13 +587,13 @@ button:hover {
   transition: 0.5s;
 }
 
-input:hover {
+/* input:hover {
   transform: translate(2.5rem, 2ex);
-}
+} */
 
-textarea:hover {
+/* textarea:hover {
   transform: translate(2.5rem, 2ex);
-}
+} */

```

Comment:
Great job team 🤝

- [ ] [Optional] Kindly remove commented code from your style sheet. This is not regarded as a good practice.